specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2021-12-01-preview/examples/ApiManagementPortalSettingsCorsPatch.json

```diff
@@ -0,0 +1,19 @@
+{
+  "parameters": {
+    "serviceName": "apimService1",
+    "resourceGroupName": "rg1",
+    "api-version": "2021-04-01-preview",
+    "subscriptionId": "subid",
+    "If-Match": "*",
+    "parameters": {
+      "properties": {
+        "allowedOrigins": [
```

Comment:
How does the patch work on `allowedOrigins`?

- Is it always expected to be append only?
- Will it be replace?

Reply:
yep, we replace the whole list.